### ISSUE TITLE
[Bug] Salvage drones bad turn rate makes them miss loot

### ISSUE THREAD

**Issue by TaipionsSalemClient** [PavelZinchenko/event-horizon-frontier-builds#288]
I see it repeatedly, that a salvage drones (sometimes 2) try to get a piece of loot and don't hit it,
because their turn rate is too bad.
They "circle" around the loot, then make a 180° turn and try again, usually failing again, only getting it after several tries.

Now that salvage drones are not part of combat anymore (they do not take dmg),
it is time to make them work properly.

- salvage drones should have perfect (instant) turn rate, like having 1 weight, to prevent such problems

These are some more issues that partly come from times back when salvage drones could be killed:

- they reset/respawn upon hitting "OK" on the mission end screen, this is not necessary anymore, and further slows the loot collection process

- salvage drones loot collection radius is too small, needs to be slightly bigger, could be a lot bigger actually

- salvage drones are too slow

- salvage drones should not interact at all with anything other than loot, not with black holes, and should not collide with anything

side note:

-  player ship on autopilot should help with collecting loot like it once did,
this should have never been removed,
right now it does not do anything useful other than drifting through space aimlessly

**Comment by Snowgods** [comment]
Personally I would like the game to instantaneously collect all loot when the level has been beaten

**Comment by PavelZinchenko** [comment]
I made the drones faster; they do their job quite well now

**Comment by TaipionsSalemClient** [comment]
> Personally I would like the game to instantaneously collect all loot when the level has been beaten

Yes, but then we would not need salvage drones and the related skill anymore,
also it adds to the game having visible loot, people tend to like that.
